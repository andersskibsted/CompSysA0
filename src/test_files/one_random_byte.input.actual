test_files/one_random_byte.input: ISO-8859 text
