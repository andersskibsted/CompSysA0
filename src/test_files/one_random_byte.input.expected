test_files/one_random_byte.input: very short file (no magic)
